apps/meteor/app/livechat/server/api/v1/mac.ts

```diff
@@ -0,0 +1,15 @@
+import { Omnichannel } from '@rocket.chat/core-services';
+
+import { API } from '../../../../api/server';
+
+API.v1.addRoute(
+	'omnichannel/mac/check',
```

Comment:
Can you implement the changes without this new endpoint? Reacting to the limitReached event instead of actively validating it every time the omnichannel is loaded?

Reply:
endp was removed in favor of `licenses.info` one, now that it has the data we need